astropy/units/quantity_helper/function_helpers.py

```diff
@@ -1264,12 +1264,12 @@ def merge_arrays(
 
     return (
         (arrays,),
-        dict(
-            fill_value=fill_value,
-            flatten=flatten,
-            usemask=usemask,
-            asrecarray=asrecarray,
-        ),
+        {
```

Comment:
Same, please exclude this one.

Reply:
This is a helper function for a numpy operation so speed is very important.
Given that, this is a good improvement.

Reply:
As above. Can "solve" the same way too if we really want to follow ruff gospel.